jenkins-jobs/job-dsl/postgresql_matrix_test.groovy

```diff
@@ -12,7 +12,7 @@ matrixJob('debezium-postgresql-matrix-test') {
          ''')
 
     axes {
-        text('POSTGRES_VERSION', '10', '11', '9.6-alpine', '10-alpine', '11-alpine')
+        text('POSTGRES_VERSION', '10', '11')
```

Comment:
Didn't we add 12 recently? And why is Alpine removed?

Reply:
@gunnarmorling Good point with 12, the changes were done in parallel so it was lost.
Regarding alpine - we've remove postgis support from them. So we need to refactor postgres tests to extract postgis tests so they can be filtered.

Reply:
> So we need to refactor postgres tests to extract postgis tests so they can be filtered.

Either that; or would it be fine to simply run the Tests against the full images? I'm torn.

Reply:
@gunnarmorling We have no full apline images and if you recall we are unable to provide them as alpine requires increaseing number of dependencis of postgis to be built from the source. That was the reason in the first case to remove postgis from alpine

Reply:
Understood. What I meant is, do we actually need to do this one:

>  So we need to refactor postgres tests to extract postgis tests so they can be filtered.

(as a follow-up). Or are we fine with this PR as is, i.e. run the test suite against the other images but not Alpine?

Reply:
I'd prefer to do the follow-up - when time permits